### PR TITLE
User runnable

### PR DESCRIPTION
**Pull request by djmitche**
@anarute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +29,7 @@ To run the provisioner locally, you will need a similar set of configuration.
 Then run
 
 ```
-npm run compile && DEBUG=* node lib/main.js development
+npm run compile && DEBUG=* node lib/main.js server
```
Comment (jhford) [inline README.md]: s/server/all/ and you can have the complete provisioner running and not just the server.  We should also do `NODE_ENV=development`

File: config.yml [hunk 2]
```diff
@@ -83,6 +83,7 @@ development:
   azure:
     account: jungle
   app:
+    awsInstancePubkey: 'fake-pubkey'
```
Comment (jhford) [inline config.yml]: i don't know if it's OK to use a fake key in this format, a valid-shaped key should be in user-config.yml instead.

File: src/main.js [hunk 3]
```diff
@@ -201,11 +201,15 @@ let load = base.loader({
...
-      });
+      if (cfg.influx.connectionString) {
+        return new base.stats.Influx({
+          connectionString: cfg.influx.connectionString,
+          maxDelay: cfg.influx.maxDelay,
+          maxPendingPoints: cfg.influx.maxPendingPoints,
+        });
+      } else {
+        console.log("No influx.connectionString configured; not using influx");
+      }
```
Comment (jhford) [inline src/main.js]: Shouldn't we use NullDrain instead?  I'm not sure if influx being undefined will be OK for the provisioning background process.

File: config.yml (hunk 2 above)
Comment (djmitche) [inline config.yml]: The field is required, but if you're not running the provisioner, it's not actually used.  So this gets the web server to run without having to create a key.

**Comment by djmitche** [comment]
Does this look better?

**Comment by jhford** [comment]
This looks fine.  I'm not merging today because it's 6:30pm and I don't want to watch the deployment today, but I will deploy it tomorrow.

**Pull request merged**